web-client/src/components/AddSockForm.tsx

```diff
@@ -0,0 +1,266 @@
+                                className="border p-2 text-black text-sm"
+                            >
+                                <option value="" disabled>
+                                    Select Size
+                                </option>
+                                {availableSizes.map((size) => (
+                                    <option
+                                        key={size}
+                                        value={size}
+                                        disabled={
+                                            selectedSizes.includes(size) && field.size !== size
+                                        }
+                                    >
+                                        {size}
+                                    </option>
+                                ))}
+                            </select>
+
+                            <Input
+                                {...register(`sizes.${index}.quantity`, {
+                                    required: "Quantity is required",
+                                    valueAsNumber: true,
+                                    min: 1,
+                                })}
+                                placeholder="Quantity"
+                                type="number"
+                                className="border p-2 text-black"
+                            />
+
+                            <Input
+                                {...register(`sizes.${index}.price`, {
+                                    required: "Price is required",
+                                    valueAsNumber: true,
+                                    min: 0.01,
+                                })}
+                                placeholder="Price"
+                                type="number"
+                                className="border p-2 text-black"
+                            />
+
+                            <Button
+                                size="icon"
+                                variant="destructive"
+                                onClick={() => remove(index)}
+                                className="h-8 w-8 flex items-center justify-center bg-red-500 border-red-500 text-white"
+                            >
+                                <Trash2 className="h-4 w-4" />
+                            </Button>
+                        </div>
+                    ))}
+
+                    <Button
+                        type="button"
+                        variant="outline"
+                        className="w-full border-black text-black"
+                        onClick={() => append({ size: "S", quantity: 1, price: 0 })}
+                    >
+                        + Add Size
+                    </Button>
+                </div>
+            )}
+
+
+            {step === 3 && (
+                <div className="space-y-4">
+                    <label className="block text-sm font-medium text-black">
```

Comment:
Update the Label styles to this:

```ts
<label className="text-sm font-medium text-muted-foreground">
  Preview Image URL
</label>
```

This is more consistent with the rest of the UI.